apps/myaccount/src/components/profile/profile.tsx

```diff
@@ -127,6 +153,25 @@ export const Profile: FunctionComponent<ProfileProps> = (props: ProfileProps): R
 
     const [ isMobileVerificationEnabled, setIsMobileVerificationEnabled ] = useState<boolean>(false);
     const [ isEmailVerificationEnabled, setIsEmailVerificationEnabled ] = useState<boolean>(false);
+    const [ isMultipleEmailAndMobileNumberEnabled, setIsMultipleEmailAndMobileNumberEnabled ] =
+        useState<boolean>(false);
+    const [ expandMultiAttributeAccordion, setExpandMultiAttributeAccordion ] = useState<Record<string, boolean>>({
+        [EMAIL_ADDRESSES_ATTRIBUTE]: false,
+        [MOBILE_NUMBERS_ATTRIBUTE]: false
+    });
+
+    // Multi-valued attribute delete confirmation modal related states.
+    const [ selectedAttributeInfo, setSelectedAttributeInfo ] =
+        useState<{ value: string; schema?: ProfileSchema }>({ value: "" });
+    const [ showMVDeleteConfirmationModal, setShowMVDeleteConfirmationModal ] = useState<boolean>(false);
+    const handleMVDeleteModalClose: () => void = useCallback(() => {
```

Comment:
let's not use arbitrary abbreviations in variable names :)

```suggestion
    const handleMultiValueAttributeDeleteConfirmModalClose: () => void = useCallback(() => {
```

Reply:
Changed